source/support.tex

```diff
@@ -4813,6 +4815,13 @@
     inline constexpr @\unspec@ compare_weak_order_fallback = @\unspec@;
     inline constexpr @\unspec@ compare_partial_order_fallback = @\unspec@;
   }
+
+  // \ref{compare.type}, type ordering
+  template<class T, class U>
+  struct type_order;
```

Comment:
indent "struct" by two

Reply:
That would be inconsistent with `struct common_comparison_category` above.  Should I indent both?

Reply:
Don't worry about existing material, let's just get the new wording right. We can clean up existing text separately if there's a pressing need.